charts/defguard/templates/defguard-config.yaml

```diff
@@ -9,7 +9,7 @@ data:
   DEFGUARD_COOKIE_DOMAIN: {{ .Values.cookie.domain }}
   {{- end }}
   DEFGUARD_COOKIE_INSECURE: {{ .Values.cookie.insecure | quote }}
-  DEFGUARD_DB_HOST: {{ include "defguard.fullname" . }}-postgresql
+  DEFGUARD_DB_HOST: {{ .Values.postgresql.host | default (printf "%s-postgresql" (include "defguard.fullname" .)) }}
   DEFGUARD_DB_NAME: {{ .Values.postgresql.auth.database }}
```

Comment:
I think for full external DB support we should also configure `DEFGUARD_DB_PORT`